_scripts/evaluate_tools.py

```diff
@@ -59,27 +59,31 @@ def get_actual_results(langchain_tracer: LangChainTracer, name: str) -> Tuple[Di
 
     if args.eval_path != "None":
         # getting eval df
-        eval_df = pd.read_csv(args.eval_path, sep=";")
+        eval_df = pd.read_csv(args.eval_path, sep=";").iloc[:2, :]
```

Comment:
Aren't we just reading the first example with this change? If so, this is really what we want or just debug/temporary changes?

Reply:
I was testing the pipeline and wanted to save on credits.. will revert :)